src/libraries/dirs.proj

```diff
@@ -1,26 +1,37 @@
 <Project>
-  <Import Project="$(MSBuildThisFileDirectory)Directory.Build.props" />
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
 
-  <!-- required to build the projects in their specified order -->
   <PropertyGroup>
-    <SerializeProjects>true</SerializeProjects>
+    <TargetFramework>$(BuildTargetFramework)</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(DirectoryToBuild)' == ''">
-    <Project Include="ref.builds" />
```

Comment:
just for curiosity, why are we converting .build to projs ?

Reply:
Yes, but he wants to control the exact time where the Sdk.targets get loaded, so he can then override the Build target. Check bellow where he actually added a comment about this.

Reply:
For Intellisense to work by default. .*proj is also restorable by nuget.